src/sitegen/ChildrenList.js

```diff
@@ -24,7 +24,7 @@ module.exports = BasePlugin.extend({
 
       _.each(files, function(file) {
          if (file.children) {
-            _.sortBy(file.children, 'title');
+            file.children = _.sortBy(file.children, 'title');
```

Comment:
This was a bug in the prototype code that I brought over from our internal codebase, and since I didn't have a way of testing this plugin at the time, I didn't catch it. Unit tests to the rescue!